docs/maintainer-notes.md

```diff
@@ -32,3 +32,35 @@ Or regenerate everything:
 ```cli
 ./generate --all
 ```
+
+## Creating a new client
+
+If you want to create a new AWS client, there are a few steps to take. Below
+is an example for DynamoDB.
+
+1. Clone the async-aws/aws repository.
+1. Run `composer install`
+1. Create `./src/DynamoDb/DynamoDbClient.php` and add an empty `DynamoDbClient` class.
+1. Update `./manifest.json`
```

Comment:
`1. Register the service in the `services` section of the `./manifest.json` file

Reply:
Updated